--- nrpe-2.12.new/sample-config/nrpe.cfg.in	2008-03-27 09:47:26.682398787 +0100
+++ nrpe-2.12/sample-config/nrpe.cfg.in	2012-01-04 17:57:22.907296034 +0200
@@ -23,7 +23,7 @@
 # number.  The file is only written if the NRPE daemon is started by the root
 # user and is running in standalone mode.
 
-pid_file=/var/run/nrpe.pid
+pid_file=/var/run/nrpe/nrpe.pid
 
 
 
@@ -169,8 +169,7 @@
 # This directive allows you to include definitions from config files (with a
 # .cfg extension) in one or more directories (with recursion).
 
-#include_dir=<somedirectory>
-#include_dir=<someotherdirectory>
+include_dir=/etc/nagios/nrpe.d
 
 
 
@@ -196,11 +195,11 @@
 
 # The following examples use hardcoded command arguments...
 
-command[check_users]=@libexecdir@/check_users -w 5 -c 10
-command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
-command[check_hda1]=@libexecdir@/check_disk -w 20% -c 10% -p /dev/hda1
-command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
+#command[check_users]=@libexecdir@/check_users -w 5 -c 10
+#command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
+#command[check_hda1]=@libexecdir@/check_disk -w 20% -c 10% -p /dev/hda1
+#command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
+#command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
 
 
 # The following examples allow user-supplied arguments and can
