diff -ur nrpe-2.12/sample-config/nrpe.cfg.in nrpe-2.12.new/sample-config/nrpe.cfg.in
--- nrpe-2.12/sample-config/nrpe.cfg.in	2007-11-23 18:31:23.000000000 +0100
+++ nrpe-2.12.new/sample-config/nrpe.cfg.in	2008-03-27 09:47:26.682398787 +0100
@@ -23,7 +23,7 @@
 # number.  The file is only written if the NRPE daemon is started by the root
 # user and is running in standalone mode.
 
-pid_file=/var/run/nrpe.pid
+pid_file=/var/run/nrpe/nrpe.pid
 
 
 
@@ -196,11 +196,11 @@
 
 # The following examples use hardcoded command arguments...
 
-command[check_users]=@libexecdir@/check_users -w 5 -c 10
-command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
-command[check_hda1]=@libexecdir@/check_disk -w 20% -c 10% -p /dev/hda1
-command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
+#command[check_users]=@libexecdir@/check_users -w 5 -c 10
+#command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
+#command[check_hda1]=@libexecdir@/check_disk -w 20% -c 10% -p /dev/hda1
+#command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
+#command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
 
 
 # The following examples allow user-supplied arguments and can
Tylko w nrpe-2.12.new/sample-config: nrpe.cfg.in~
