--- nrpe-3.2.1/sample-config/nrpe.cfg.in~	2017-09-01 15:59:54.000000000 +0200
+++ nrpe-3.2.1/sample-config/nrpe.cfg.in	2018-09-20 14:02:57.989524019 +0200
@@ -270,92 +270,10 @@ connection_timeout=300
 
 # nasty_metachars="|`&><'\\[]{};\r\n"
 
-
-
-# COMMAND DEFINITIONS
-# Command definitions that this daemon will run.  Definitions
-# are in the following format:
-#
-# command[<command_name>]=<command_line>
-#
-# When the daemon receives a request to return the results of <command_name>
-# it will execute the command specified by the <command_line> argument.
-#
-# Unlike Nagios, the command line cannot contain macros - it must be
-# typed exactly as it should be executed.
-#
-# Note: Any plugins that are used in the command lines must reside
-# on the machine that this daemon is running on!  The examples below
-# assume that you have plugins installed in a /usr/local/nagios/libexec
-# directory.  Also note that you will have to modify the definitions below
-# to match the argument format the plugins expect.  Remember, these are
-# examples only!
-
-
-# The following examples use hardcoded command arguments...
-# This is by far the most secure method of using NRPE
-
-command[check_users]=@pluginsdir@/check_users -w 5 -c 10
-command[check_load]=@pluginsdir@/check_load -r -w .15,.10,.05 -c .30,.25,.20
-command[check_hda1]=@pluginsdir@/check_disk -w 20% -c 10% -p /dev/hda1
-command[check_zombie_procs]=@pluginsdir@/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=@pluginsdir@/check_procs -w 150 -c 200
-
-
-# The following examples allow user-supplied arguments and can
-# only be used if the NRPE daemon was compiled with support for
-# command arguments *AND* the dont_blame_nrpe directive in this
-# config file is set to '1'.  This poses a potential security risk, so
-# make sure you read the SECURITY file before doing this.
-
-### MISC SYSTEM METRICS ###
-#command[check_users]=@pluginsdir@/check_users $ARG1$
-#command[check_load]=@pluginsdir@/check_load $ARG1$
-#command[check_disk]=@pluginsdir@/check_disk $ARG1$
-#command[check_swap]=@pluginsdir@/check_swap $ARG1$
-#command[check_cpu_stats]=@pluginsdir@/check_cpu_stats.sh $ARG1$
-#command[check_mem]=@pluginsdir@/custom_check_mem -n $ARG1$
-
-### GENERIC SERVICES ###
-#command[check_init_service]=sudo @pluginsdir@/check_init_service $ARG1$
-#command[check_services]=@pluginsdir@/check_services -p $ARG1$
-
-### SYSTEM UPDATES ###
-#command[check_yum]=@pluginsdir@/check_yum
-#command[check_apt]=@pluginsdir@/check_apt
-
-### PROCESSES ###
-#command[check_all_procs]=@pluginsdir@/custom_check_procs
-#command[check_procs]=@pluginsdir@/check_procs $ARG1$
-
-### OPEN FILES ###
-#command[check_open_files]=@pluginsdir@/check_open_files.pl $ARG1$
-
-### NETWORK CONNECTIONS ###
-#command[check_netstat]=@pluginsdir@/check_netstat.pl -p $ARG1$ $ARG2$
-
-### ASTERISK ###
-#command[check_asterisk]=@pluginsdir@/check_asterisk.pl $ARG1$
-#command[check_sip]=@pluginsdir@/check_sip $ARG1$
-#command[check_asterisk_sip_peers]=sudo @pluginsdir@/check_asterisk_sip_peers.sh $ARG1$
-#command[check_asterisk_version]=@pluginsdir@/nagisk.pl -c version
-#command[check_asterisk_peers]=@pluginsdir@/nagisk.pl -c peers
-#command[check_asterisk_channels]=@pluginsdir@/nagisk.pl -c channels 
-#command[check_asterisk_zaptel]=@pluginsdir@/nagisk.pl -c zaptel 
-#command[check_asterisk_span]=@pluginsdir@/nagisk.pl -c span -s 1
-
-
-
-# INCLUDE CONFIG FILE
-# This directive allows you to include definitions from an external config file.
-
-#include=<somefile.cfg>
-
-
-
 # INCLUDE CONFIG DIRECTORY
 # This directive allows you to include definitions from config files (with a
 # .cfg extension) in one or more directories (with recursion).
 
 #include_dir=<somedirectory>
 #include_dir=<someotherdirectory>
+include_dir=/etc/nagios/nrpe.d
