--- nrpe-2.4/sample-config/nrpe.cfg.in~	2006-02-04 00:02:32.000000000 +0200
+++ nrpe-2.4/sample-config/nrpe.cfg.in	2006-05-21 22:20:00.671748952 +0300
@@ -162,12 +162,12 @@
 
 # The following examples use hardcoded command arguments...
 
-command[check_users]=@libexecdir@/check_users -w 5 -c 10
-command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
-command[check_disk1]=@libexecdir@/check_disk -w 20 -c 10 -p /dev/hda1
-command[check_disk2]=@libexecdir@/check_disk -w 20 -c 10 -p /dev/hdb1
-command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
+#command[check_users]=@libexecdir@/check_users -w 5 -c 10
+#command[check_load]=@libexecdir@/check_load -w 15,10,5 -c 30,25,20
+#command[check_disk1]=@libexecdir@/check_disk -w 20 -c 10 -p /dev/hda1
+#command[check_disk2]=@libexecdir@/check_disk -w 20 -c 10 -p /dev/hdb1
+#command[check_zombie_procs]=@libexecdir@/check_procs -w 5 -c 10 -s Z
+#command[check_total_procs]=@libexecdir@/check_procs -w 150 -c 200 
 
 # The following examples allow user-supplied arguments and can
 # only be used if the NRPE daemon was compiled with support for 
