--- nrpe-3.0.orig/sample-config/nrpe.cfg.in	2016-07-12 23:24:40.000000000 +0300
+++ nrpe-3.0/sample-config/nrpe.cfg.in	2016-09-07 15:46:35.913180521 +0300
@@ -23,7 +23,7 @@
 # number.  The file is only written if the NRPE daemon is started by the root
 # user and is running in standalone mode.
 
-pid_file=@piddir@/nrpe.pid
+pid_file=/var/run/nrpe/nrpe.pid
 
 
 
@@ -257,47 +257,8 @@
 # This directive allows you to include definitions from config files (with a
 # .cfg extension) in one or more directories (with recursion).
 
-#include_dir=<somedirectory>
-#include_dir=<someotherdirectory>
+include_dir=/etc/nagios/nrpe.d
 
 
-
-# COMMAND DEFINITIONS
-# Command definitions that this daemon will run.  Definitions
-# are in the following format:
-#
-# command[<command_name>]=<command_line>
-#
-# When the daemon receives a request to return the results of <command_name>
-# it will execute the command specified by the <command_line> argument.
-#
-# Unlike Nagios, the command line cannot contain macros - it must be
-# typed exactly as it should be executed.
-#
-# Note: Any plugins that are used in the command lines must reside
-# on the machine that this daemon is running on!  The examples below
-# assume that you have plugins installed in a /usr/local/nagios/libexec
-# directory.  Also note that you will have to modify the definitions below
-# to match the argument format the plugins expect.  Remember, these are
-# examples only!
-
-
-# The following examples use hardcoded command arguments...
-
-command[check_users]=@pluginsdir@/check_users -w 5 -c 10
-command[check_load]=@pluginsdir@/check_load -w 15,10,5 -c 30,25,20
-command[check_hda1]=@pluginsdir@/check_disk -w 20% -c 10% -p /dev/hda1
-command[check_zombie_procs]=@pluginsdir@/check_procs -w 5 -c 10 -s Z
-command[check_total_procs]=@pluginsdir@/check_procs -w 150 -c 200
-
-
-# The following examples allow user-supplied arguments and can
-# only be used if the NRPE daemon was compiled with support for
-# command arguments *AND* the dont_blame_nrpe directive in this
-# config file is set to '1'.  This poses a potential security risk, so
-# make sure you read the SECURITY file before doing this.
-
-#command[check_users]=@pluginsdir@/check_users -w $ARG1$ -c $ARG2$
-#command[check_load]=@pluginsdir@/check_load -w $ARG1$ -c $ARG2$
-#command[check_disk]=@pluginsdir@/check_disk -w $ARG1$ -c $ARG2$ -p $ARG3$
-#command[check_procs]=@pluginsdir@/check_procs -w $ARG1$ -c $ARG2$ -s $ARG3$
+# NOTE:
+# In PLD Linux, command definitions are in file /etc/nagios/nrpe.d/commands.cfg
